projects/isolate_proto/src/isolate_proto/health/health_pb2_grpc.py

```diff
@@ -1,10 +1,33 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 """Client and server classes corresponding to protobuf-defined services."""
-from __future__ import annotations
-
 import grpc
-
-from isolate_proto.health import health_pb2 as health__pb2
+import warnings
+
+from isolate.server.health import health_pb2 as health__pb2
+
+GRPC_GENERATED_VERSION = '1.64.0'
```

Comment:
After so many years grpc finally figured out that they can do this... Better late than never.